doc/build-unix.md

```diff
@@ -107,25 +107,21 @@ ZMQ dependencies (provides ZMQ API 4.x):
 Dependencies for the GUI: Ubuntu & Debian
 -----------------------------------------
 
+Merit is configured without building GUI by default.
 If you want to build Merit-Qt, make sure that the required packages for Qt development
 are installed. Either Qt 5 or Qt 4 are necessary to build the GUI.
-If both Qt 4 and Qt 5 are installed, Qt 5 will be used. Pass `--with-gui=qt4` to configure to choose Qt4.
-To build without GUI pass `--without-gui`.
 
 To build with Qt 5 (recommended) you need the following:
 
-    sudo apt-get install libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler libarchive-dev
+    sudo apt-get install libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler libarchive-dev libqrencode-dev
 
 Alternatively, to build with Qt 4 you need the following:
 
-    sudo apt-get install libqt4-dev libprotobuf-dev protobuf-compiler libarchive-dev
+    sudo apt-get install libqt4-dev libprotobuf-dev protobuf-compiler libarchive-dev libqrencode-dev
```

Comment:
Do we have these libraries in other build descriptions? (libarchive and libqrencode)